package.json

```diff
@@ -47,7 +47,7 @@
     "prop-types": "15.8.1",
     "react": "16.14.0",
     "react-dom": "16.14.0",
-    "react-instantsearch-hooks-web": "^6.40.1",
+    "react-instantsearch": "^7.0.2",
```

Comment:
Should we remove the `^` and allow renovate to update this dependency?

Reply:
Oh, I didn't immediately pick up that we were moving to another package too. What is the difference between the two?

Reply:
From what I gather, there are a lot of different versions of this package. Some of the APIs have been deprecated, but none of these are being used by this application. The APIs that haven't been removed remain unchanged, only the name of the package is different. I think this is part of an effort to make the overall experience less confusing. Their doc site is a little bit of a maze with all of the different (but very similar) names for things.

https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/react/#migrate-from-react-instantsearch-hooks-to-react-instantsearch-v7